hibernate-core/src/main/java/org/hibernate/dialect/CockroachDialect.java

```diff
@@ -181,13 +186,8 @@ protected void registerColumnTypes(TypeContributions typeContributions, ServiceR
 		ddlTypeRegistry.addDescriptor( new Scale6IntervalSecondDdlType( this ) );
 
 		// Prefer jsonb if possible
-		if ( getVersion().isSameOrAfter( 20 ) ) {
-			ddlTypeRegistry.addDescriptor( new DdlTypeImpl( INET, "inet", this ) );
-			ddlTypeRegistry.addDescriptor( new DdlTypeImpl( JSON, "jsonb", this ) );
-		}
-		else {
-			ddlTypeRegistry.addDescriptor( new DdlTypeImpl( JSON, "json", this ) );
-		}
```

Comment:
wasn't sure about removing this if, since it didn't include the minor. But afaict the inet data type was supported already before that anyhow.

Reply:
If anyone complains about that, we can still add that later.